services/packagist/packagist-version.tester.js

```diff
@@ -22,6 +22,14 @@ t.create('version (valid)')
     message: isPackagistVersion,
   })
 
+t.create('version (no releases)')
+  .get('/v/wsg/hello.json')
```

Comment:
This example did trigger the error reported in #3709, `Can't generate a version color for undefined`, even though it's a [perfectly valid package](https://packagist.org/packages/wsg/hello) with one version item returned in the [API response](https://packagist.org/packages/wsg/hello.json).